canonical/multipass
# Release 1.6.0

Test platforms:
- [x] Ubuntu@qemu (Ricardo)
- [x] Ubuntu@LXD (Chris)
- [x] Ubuntu@libvirt (Ricardo)
- [x] macOS@hyperkit (Saviq)
- [x] macOS@VirtualBox (Chris)
  - Found #1951 
- [x] Windows@Hyper-V (Luis)
- [x] Windows@VirtualBox (Luis)

Test plan:
1. install previous release
2. launch a couple instances (including `primary`)
3. update to the RC, verify instances around
4. uninstall / purge and install new
5. launch instances (with, and without `--network`)
   - `primary`
   - older Ubuntu
   - Ubuntu Core
   - file / http
